pubsub/google/cloud/pubsub_v1/types.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import collections
+import sys
+
+import psutil
+
+from google.cloud.proto.pubsub.v1 import pubsub_pb2
+from google.gax.utils.messages import get_messages
+
+
+# Define the default values for batching.
+#
+# This class is used when creating a publisher or subscriber client, and
+# these settings can be altered to tweak Pub/Sub behavior.
+# The defaults should be fine for most use cases.
+BatchSettings = collections.namedtuple('BatchSettings',
+    ['max_bytes', 'max_latency', 'max_messages'],
+)
+BatchSettings.__new__.__defaults__ = (
```

Comment:
I am having trouble understanding from that code what the defaults _are_. Happy to change them though.